bootstrap/README

```diff
@@ -2,6 +2,6 @@ This directory contains scripts that install necessary OS-specific
 prerequisite dependencies (see docs/using.rst).
 
 General dependencies:
-- git-core: py26reqs.txt git+https://*
```

Comment:
if this is removed, then `git-core` should be removed as well, or the comment should be updated if, some other projects are downloaded using `pip install git+http...`